Endzone.Umbraco.Extensions/PublishedContentExtensions/Collections.cs

```diff
@@ -89,5 +104,18 @@ public static IEnumerable<IEnumerable<T>> Split<T>(this IEnumerable<T> list, int
                          select part.AsEnumerable();
             return splits;
         }
+
+        /// <summary>
+        /// Returns the website settings node
+        /// </summary>
+        /// <param name="content"></param>
+        /// <returns></returns>
+        public static IPublishedContent GetWebsiteSettings(this IPublishedContent content)
```

Comment:
Yeah, if you think we should be using it then I am happy with that.

Yes, split it into a separate class - Configuration sounds good to me